SFR SEFOR model refs #<690> (#719)
idaholab/virtual_test_bed: doc/content/sfr/sefor/index.md

# Southwest Experimental Fast Oxide Reactor (SEFOR) Numerical Models

*Contacts: Donny Hartanto (hartantod.at.ornl.gov)*, Ahmed Amin E. Abdelhameed (aabdelhameed.at.anl.gov)*, Yan Cao  (ycao.at.anl.gov)*, Eva Davidson (davidsonee.at.ornl.gov), Emily Shemon (eshemon.at.anl.gov)

*Model link: [SEFOR](https://github.com/idaholab/virtual_test_bed/tree/devel/sfr/sefor)*

!tag name=Southwest Experimental Fast Oxide Reactor (SEFOR)

>      description=Simulation a series of SEFOR Core using the NEAMS tools pairs=reactor_type:sfr

Reviewer: ```suggestion
     description=Simulation of SEFOR II and IE Core experiments using the NEAMS tools=reactor_type:sfr
```

           reactor:SEFOR
           geometry:core
           simulation_type:neutronics

>            input_features:reactor_meshing;shift_reference;cross_section_generation;Griffin_neutronics

Author: Got it and fixed
